packages/components/src/InputDate/useInputDateActivatorActions.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent, FocusEvent, useState } from "react";
+import { InputDateRebuiltProps } from "./InputDate.types";
+import { DatePickerActivatorProps } from "../DatePicker/DatePickerActivator";
+
+export interface useInputDateActivatorActionsProps
+  extends Pick<InputDateRebuiltProps, "onFocus" | "onBlur"> {
+  onChange: DatePickerActivatorProps["onChange"];
+}
+
+/**
+ * Combines the actions on the InputDate such as onChange, onFocus, onBlur to forward event handler passed to the InputDate component to the DatePicker component.
+ * DO not repeat this pattern. We are doing this as a proof of concept relating to the refactoring of Form inputs to see what can be removed.
+ */
+export function useInputDateActivatorActions({
+  onChange,
+  onBlur,
+  onFocus,
+}: useInputDateActivatorActionsProps) {
+  const [isFocused, setIsFocused] = useState(false);
+
+  function handleChange(
+    _: unknown,
```

Comment:
Can you give more context for the `unknown`?
Can we get away with removing that or on the flip side, being more explicit about the value type?

Reply:
Since it was an unused parameter I just decided to use unknown because it is used to provide the change event to the InputDateActivator